src/patternfly/components/Toolbar/toolbar.scss

```diff
@@ -9,13 +9,18 @@ $pf-c-toolbar--inset-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl"
   --pf-c-toolbar--RowGap: var(--pf-global--spacer--lg);
   --pf-c-toolbar--PaddingTop: var(--pf-global--spacer--md);
   --pf-c-toolbar--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-toolbar--AlignItems--base: baseline;
+  --pf-c-toolbar--AlignSelf--base: auto;
```

Comment:
I wonder if we should have different `--base` vars for the things we allow alignments for. Just because some things may not have a default of `align-items: baseline` or `align-self: auto`. I'm also not 100% on why we have `--base` vars since we don't have responsive support where you want to reference/return to whatever the default is with a modifier. If we introduced that and needed `--base` vars, we could do that without a breaking change.